notebooks/example_create_agent.ipynb

```diff
@@ -0,0 +1,284 @@
+    "\n",
+    "## References\n",
+    "- [Langchain Agent Docs](https://python.langchain.com/docs/modules/agents/)\n",
+    "- [Gen AI Agent Orchestrator](https://docs.glair.ai/generative-internal/modules/agent-orchestrator)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare Environment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Install Dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install google-search-results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Install Gen AI SDK\n",
+    "\n",
+    "To install the SDK library, we need to create a personal access token on GitHub. Please follow these steps:\n",
+    "1. Log in to your [GitHub Account](https://github.com/).\n",
+    "2. Go to the [Personal Access Tokens](https://github.com/settings/tokens) page.\n",
+    "3. If you haven't created a Personal Access Tokens yet, you can generate one.\n",
+    "4. When generating a new token, make sure that you have checked the `repo` option to grant access to private repositories.\n",
+    "5. Now, you can copy the new token that you have generated and paste it into the script below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import getpass\n",
+    "import subprocess\n",
+    "import sys\n",
+    "import os\n",
+    "\n",
+    "def install_sdk_library():\n",
+    "    token = getpass.getpass(\"Input Your Personal Access Token: \")\n",
+    "\n",
+    "    cmd = f\"pip install -e git+https://{token}@github.com:GDP-ADMIN/gen-ai-internal.git#egg=gdplabs_gen_ai\"\n",
+    "\n",
+    "    with subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True) as process:\n",
+    "        stdout, stderr = process.communicate()\n",
+    "\n",
+    "        if process.returncode != 0:\n",
+    "            sys.stdout.write(stderr)\n",
+    "            raise subprocess.CalledProcessError(returncode=process.returncode, cmd=cmd)\n",
+    "        else:\n",
+    "            sys.stdout.write(stdout)\n",
+    "\n",
+    "install_sdk_library()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Add Environment Variable\n",
+    "\n",
+    "You will need two provide two API Keys. OpenAI API Key and SerpAPI API Key."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "os.environ[\"OPENAI_API_KEY\"] = getpass.getpass(\"OPENAI API KEY: \")\n",
+    "os.environ[\"SERPAPI_API_KEY\"] = getpass.getpass(\"SERPERAI API KEY: \")"
```

Comment:
Please add an explanation of  url to the serper like this: https://serpapi.com/dashboard
THis to make sure people know it directly